src/transactions/transactionBytes.js

```diff
@@ -12,181 +12,126 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-/* eslint-disable no-plusplus */
-import ByteBuffer from 'bytebuffer';
 import bignum from 'browserify-bignum';
 
+const byteSizes = {
+	TYPE: 1,
+	TIMESTAMP: 4,
+	MULTISIGNATURE_PUBLICKEY: 32,
+	RECIPIENT_ID: 8,
+	AMOUNT: 8,
+	SIGNATURE_TRANSACTION: 64,
+	SECOND_SIGNATURE_TRANSACTION: 64,
+	DATA: 64,
+};
+
 /**
- * @method getEmptyBytesForTransaction
- * @param transaction Object
- * @return {object}
+ * @method getAssetBytesHelper
+ * @return {Buffer}
  */
 
-function getEmptyBytesForTransaction(transaction) {
+function getAssetBytesHelper(transaction) {
 	/**
 	 * @method isSendTransaction
-	 * @return {object}
+	 * @return {Buffer}
 	 */
 
 	function isSendTransaction() {
```

Comment:
Can we extract these outside `getAssetBytesHelper` into self-contained, pure functions for readability?